src/client/config.go

```diff
@@ -0,0 +1,13 @@
+package client
+
+import (
+    "k8s.io/client-go/rest"
+    "k8s.io/client-go/tools/clientcmd"
+)
+
+func BuildRESTConfig() (*rest.Config, error) {
+    rules := clientcmd.NewDefaultClientConfigLoadingRules()
```

Comment:
do you happen to know if this method respects the `KUBECONFIG` environment variable? In other words, if I have a kubectl config at `$HOME/.kube/config` but also have the env var `KUBECONFIG=project/kubeconfig` will it take context and such from the latter config?

Reply:
Yes, see here: https://github.com/kubernetes/client-go/blob/ee7a1ba5cdf1292b67a1fdf1fa28f90d2a7b0084/tools/clientcmd/loader.go#L137